rMATS_pipeline/rmatspipeline/rmatspipeline.pyx

```diff
@@ -2971,9 +2997,14 @@ cdef size_t _load_job(str rmatsf, list vbams, list flags, const int& rl,
         # processing junction reads
         for i in range(len(bams)):
             num = int(fp.readline())
-            idx = vbams.index(bams[i])
+            idx = try_get_index(vbams, bams[i], &index_found)
+
```

Comment:
I don't think 'idx = try_get_index(vbams, bams[i], &index_found)' here is necessary. Since this sanity check has already been performed in the first for loop.